docs/production-deployment/cloud/nexus/index.mdx

```diff
@@ -0,0 +1,87 @@
+---
+id: index
+slug: /cloud/nexus
+title: Temporal Nexus
+description: Discover Temporal Nexus, a powerful feature for connecting durable executions across team, namespace, region, and cloud boundaries. Nexus also enables each team to have their own namespace for improved security, troubleshooting, and blast radius isolation.
+sidebar_label: Temporal Nexus 
+tags:
+  - temporal cloud
+  - temporal nexus
+keywords:
+  - Temporal Nexus
+  - cross-namespace workflows
+  - modular workflow design
+  - workflow orchestration
+  - Temporal Cloud security
+  - Temporal Cloud services
+  - distributed workflows
+  - workflow API management
+---
+
+:::tip SUPPORT, STABILITY, and DEPENDENCY INFO
+
+Temporal Nexus is in [Public Preview](https://docs.temporal.io/evaluate/development-production-features/release-stages\#public-preview) for Temporal Cloud.
+
+:::
+
+This Temporal Nexus guide covers the following topics:
+
+- [Introduction to Nexus](#nexus-introduction)  
+- [Using Nexus](/cloud/nexus/using-nexus)  
+- [Pricing](/cloud/nexus/pricing)  
+- [Getting Started](/cloud/nexus/getting-started)  
+- [Operations](/cloud/nexus/operations)  
+
+
+## Introduction to Nexus {#nexus-introduction}
+Temporal Nexus is a feature of the Temporal platform designed to connect durable executions across team, Namespace, region, and cloud boundaries. 
+It promotes a more modular architecture for sharing a subset of your team’s capabilities through well-defined service API contracts for other teams to use, that abstract underlying Temporal primitives, like Workflows, or execute arbitrary code.
+
+**Why use Temporal Nexus?**
+
+* Fully integrated Temporal SDK experience:  
+  * Run Nexus Services in a Worker, often the same Worker as underlying primitives.  
+  * Use Nexus Services from a Workflow.  
+  * Same queue-based Worker architecture, for Worker fleet load balancing.  
+* Built-in Nexus machinery:
+  * Connects Nexus callers and handlers through Nexus Endpoints.  
+  * Automatic retries & circuit breaking. 
+* Multi-region routing across Namespaces:
+  * Secure mTLS connectivity. 
+  * Endpoint access control.  
+* Observability:
+  * Integrated into Workflow Event History.  
+  * Bi-direction linking for end-to-end debugging.  
+  * Metrics, traces, and logs. 
+* Improved blast radius isolation: 
+  * Each team can have it’s own Namespace.  
+  * Per-Namespace security & rate limits.  
+  * Easier troubleshooting with less noise.
+
+**Should you be using Temporal Nexus?**
+
+* **Modular design** \- Temporal Nexus enables a modular design that helps structure an application for fast growth and supports teams working on their own parts independently. 
+If you want to decouple callers from handlers to enable moving a service to a different Namespace or Task Queue later, Nexus can encapsulate those implementation details to avoid breaking changes and cross-team coordination.
```

Comment:
```suggestion
  If you want to decouple callers from handlers to enable moving a service to a different Namespace or Task Queue later, Nexus can encapsulate those implementation details to avoid breaking changes and cross-team coordination.
```